src/components/AddItems.js

```diff
@@ -36,6 +42,25 @@ const AddItemsToList = () => {
       .get()
       .then((data) => {
         if (data.docs.length) {
+          const normalizeString = (str) => {
+            const nonWordCharactersAndUnderscores = /[\W_]/g;
```

Comment:
Wow! Great job with implementing such a clean and powerful regex here! Works like a charm 🙌